veg2hab/vegkartering.py

```diff
@@ -1215,12 +1215,16 @@ def check_mitsen(self, fgr: FGR, bodemkaart: Bodemkaart, lbk: LBK) -> None:
 
         ### Verrijken met de benodigde informatie
         if fgr_needed.any():
-            mits_info_df["fgr"] = fgr.for_geometry(mits_info_df.loc[fgr_needed])
+            mits_info_df = mits_info_df.join(
```

Comment:
deze change begrijp ik niet.

Reply:
Uit for_geometry komt nu niet langer een Series met 1 kolom (fgr of lbk bv), maar een GDF met 2 kolommon (fgr+fgr_overlap of lbk+lbk_overlap). Ik heb het nu ook even beter in de sjoin_largest_overlap docstring gezet en de typehinting daar aangepast.